src/dags/common/__init__.py

```diff
@@ -69,3 +70,22 @@ def slack_failed_task(context):
 
 vsd_default_args = default_args.copy()
 vsd_default_args["postgres_conn_id"] = "postgres_default"
+
+
+def addloopvariables(iterable):
+    """ Pass through all values from the given iterable.
```

Comment:
Ik zou dit zelf anders doen met l = list(iterable) 
for i, item in enumerate(l):
  if l == 0:
    first = Treu:
  elif l == len(l)
    last = True

Dat vind ik duidelijker

Reply:
Ja, deze is denk ik wat overkill hier. In dit geval kan deze hele functie eigenlijk helemaal weg en met een eenvoudig if-je op de plek waar deze nu wordt aangeroepen.
Overigens, voor hele grote lijsten zou het wel inefficient worden om de hele list te 'unpacken' met 'list()' aanroep.

Reply:
Ok, als het alleen gaat om indicatie first en last, zou je ook iets van short circuiting kunnen doen:

`
def voorbeeld():
    list = [1,2,3,4,5]
    first = list[0] and (list[0], True, False)
    last = list[-1] and (list[-1], True, False)
    yield first
    yield last
`